lib/spirv/lib/src/transpiler.dart

```diff
@@ -504,6 +535,20 @@ class _Transpiler {
     functionTypes[id] = _FunctionType(returnType, params);
   }
 
+  void opConstantTrue() {
+    final int type = readWord();
+    final String id = resolveName(readWord());
+    final String typeName = resolveType(type);
+    src.writeln('const $typeName $id = true;');
```

Comment:
instead of defining a constant in the output, I think the transpiler should just keep track of the ids for constantTrue and constantFalse, and always resolve their names directly to 'true' or 'false' in the resolveName function

Reply:
That makes sense, I will update